account_constraints/model/account_move_line.py

```diff
@@ -28,7 +28,8 @@ class AccountMoveLine(models.Model):
     def _get_write_authorized_fields(self):
         """ This method can be overrride to add some field that can be written
         directly on account move line """
-        return ["reconcile_id", "reconcile_partial_id"]
+        return ["reconcile_id", "reconcile_partial_id", "followup_line_id",
```

Comment:
Are these fields included in the account module or we should depend now of account_followup?